content/articles/ordering-a-wildcard-certificate.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Ordering a Wildcard SSL Certificate
+excerpt: How to order a wildcard SSL certificate with DNSimple.
+categories:
+- SSL Certificates
+---
+
+# Ordering a Standard SSL Certificate
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Wildcard certificates allow you to secure all subdomains of a domain with a single certificate. Wildcard certificates can make certificate management easier in some cases. We still recommend non-wildcard certificates for most use cases.
+
+Wildcard certificates are no different to standard certificates, they only contain an `*` as the subdomain, so please make sure to check out the article about [Ordering a Standard Certificate](/articles/ordering-standard-certificate/) too.
+
+
+## Let's Encrypt or Standard
+
+Both providers support wildcard certificates Let's Encrypt and Comodo. Let's Encrypt certificates are free but are only valid for 60 days, Comodo certificates are valid for one year or longer but also cost $100 per wildcard certificate.
+
+You can learn more about the differences between standard and Let's Encrypt certificates in the corresponding [article](/articles/standard-vs-letsencrypt/).
+
+<info>
+Let's Encrypt wildcard certificates are only available on [specific plans](https://dnsimple.com/pricing).
+</info>
+
+## Use * for a wildcard certificate
```

Comment:
This section is the core of the article, and it's what I think should contain the specific information. The way you request a wildcard is different between Standard and Let's Encrypt.

What I'd like to see is one section per certificate type where you say:

- Here's how to order a wildcard for Let's Encrypt (linking to https://support.dnsimple.com/articles/ordering-lets-encrypt-certificate/)
- Here's how to order a wildcard for Standard (linking to https://support.dnsimple.com/articles/ordering-standard-certificate/)

The instructions here are very generic and confusing. They don't provide clear direction on both cases.